integration/env/new_azure_go_file_to_improve_the_diff_should_rplace_azure_go.go

```diff
@@ -0,0 +1,125 @@
+package env
```

Comment:
This is a new file improve the diff. This is meant to replace `azure.go` in the end.